README.md

```diff
@@ -14,7 +14,7 @@ exercises folder, there is always an index file linking to all the available exe
 topic [conditionals](/notebooks/30_conditionals). The folder contains the following files:
 
 - [conditionals.ipynb](/notebooks/30_conditionals/conditionals.ipynb): The introduction to the topic (i.e. the lecture)
-- [conditionals_howto.ipynb](/notebooks/30_conditionals/conditionals_howto.ipynb): the howto 
+- [conditionals_howto.ipynb](/notebooks/30_conditionals/conditionals_howto_eng.ipynb): the howto 
```

Comment:
Why do you only link to the English howto? I guess we need a separate folder structure for the translated files. Either on the top level of the repository de and en. Or within each topic de and en. 
What do you think would work better?

Reply:
I think it's a good idea.
I will do some change later after finishing translation